src/jti_background.js

```diff
@@ -77,6 +78,30 @@ function getData (callback) {
     });
 }
 
+// Return the "rate it now" flag
+function getRateStatus (callback) {
+    chrome.storage.sync.get(rateStatus, function (status) {
+        if (status[rateStatus]) {
+            callback(true, '', status[rateStatus]);
+        } else {
+            callback(false, 'No data is currently loaded');
+        }
+    });
+}
+
+// Set the "rate it now" flag to be true
+function setRateStatus (callback) {
```

Comment:
Because I think we should keep all our data under the one "ID" I think you'll need to update this function to first fetch all the data ->
`chrome.storage.sync.get(StorageID, function (data) {....` then update the rateStatus -> `data[storageTogglesKey][rateStatus] = true'
then save the data back to the storage.

I think you could potentially repurpose the "saveTemplates" to something more generic like saveData.